### ISSUE TITLE
KeyboardInterrupt Stopping your-app ... /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-compose-0.8.3/lib/docker/compose/session.rb:201: warning: conflicting chdir during another chdir block Stopping your-app ... done

### ISSUE THREAD

**Issue by EugenMayer** [xeger/docker-compose#37]
when e.g. using `docker-sync-stack start` on the boilerplate here https://github.com/EugenMayer/docker-sync-boilerplate/tree/master/unison and then `CTRL+C` i get 

```
Traceback (most recent call last):
  File "/usr/local/bin/unison-fsmonitor", line 272, in <module>
/opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-sync-0.1.2/lib/docker-sync/execution.rb:51:in ``': Interrupt
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-sync-0.1.2/lib/docker-sync/execution.rb:51:in `block in forkexec'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-sync-0.1.2/lib/docker-sync/execution.rb:50:in `fork'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-sync-0.1.2/lib/docker-sync/execution.rb:50:in `forkexec'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-sync-0.1.2/lib/docker-sync/sync_strategy/unison.rb:71:in `watch'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-sync-0.1.2/lib/docker-sync/watch_strategy/unison.rb:27:in `run'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-sync-0.1.2/lib/docker-sync/sync_process.rb:78:in `run'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-sync-0.1.2/lib/docker-sync/sync_manager.rb:121:in `block in run'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-sync-0.1.2/lib/docker-sync/sync_manager.rb:120:in `each'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-sync-0.1.2/lib/docker-sync/sync_manager.rb:120:in `run'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-sync-0.1.2/tasks/stack/stack.thor:41:in `start'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
       	from /opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-sync-0.1.2/bin/docker-sync-stack:26:in `<top (required)>'
       	from /opt/boxen/rbenv/versions/2.2.4/bin/docker-sync-stack:23:in `load'
       	from /opt/boxen/rbenv/versions/2.2.4/bin/docker-sync-stack:23:in `<main>'
    main()
  File "/usr/local/bin/unison-fsmonitor", line 216, in main
    [cmd, args] = recvCmd();
  File "/usr/local/bin/unison-fsmonitor", line 98, in recvCmd
    line = sys.stdin.readline()
KeyboardInterrupt
Stopping your-app ...
/opt/rubies/2.2.4/lib/ruby/gems/2.2.0/gems/docker-compose-0.8.3/lib/docker/compose/session.rb:201: warning: conflicting chdir during another chdir block
Stopping your-app ... done
```

is there anything we can do about `warning: conflicting chdir during another chdir block` ?

This is how i call stop on compose: https://github.com/EugenMayer/docker-sync/blob/master/lib/docker-sync/compose.rb#L50 and thats how it is initialized https://github.com/EugenMayer/docker-sync/blob/master/lib/docker-sync/compose.rb#L50
and that is how the debootstrap is triggered on CTRL+c https://github.com/EugenMayer/docker-sync/blob/master/tasks/stack/stack.thor#L55

**Comment by xeger** [comment]
@EugenMayer, I fixed this on the master branch by adding a feature to backticks. Published as 1.0.0rc3; let me know how it works.

**Comment by EugenMayer** [comment]
Thats amazing! will try
